src/BenchmarksApps/TodosApi/HostEnvironmentExtensions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Microsoft.Extensions.Hosting;
+
+internal static class HostEnvironmentExtensions
+{
+    public static bool IsBuild(this IHostEnvironment hostEnvironment) => hostEnvironment.IsEnvironment("Build");
```

Comment:
Where does this `"Build"` environment get defined/set?

Reply:
You have to set it manually via the `ASPNETCORE_ENVIRONMENT` env var. So e.g. in PowerShell to generate the swagger file you'd do this:

```PowerShell
> $env:ASPNETCORE_ENVIRONMENT='Build' ; dotnet build -p:OpenApiGenerateDocumentsOnBuild=true ; $env:ASPNETCORE_ENVIRONMENT=$null
```